pkgs/by-name/gn/gnucobol/package.nix

```diff
@@ -0,0 +1,141 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  # Using autoconf 2.69 as that's the targeted version by upstream
+  autoconf269,
+  automake,
+  libtool,
+  # libs
+  cjson,
+  db,
+  gmp,
+  libxml2,
+  ncurses,
+  # docs
+  help2man,
+  texinfo,
+  texliveBasic,
+  # test
+  perl,
+}:
+
+let
+  nistTestSuite = stdenv.mkDerivation {
+    # This derivation is used to check GnuCOBOL with the NIST test suite
+    pname = "gnucobol-nist-test-suite";
+    # Date from curl -I
+    version = "unstable-2016-05-01";
+
+    src = fetchurl {
+      url = "mirror://sourceforge/gnucobol/newcob.val.tar.gz";
+      hash = "sha256-5FE/JqmziRH3v4gv49MzmoC0XKvCyvheswVbD1zofuA=";
+    };
+
+    # Don't unpack the file. GnuCOBOL's `make test` will take care of it
+    dontUnpack = true;
+    installPhase = ''
+      mkdir $out
+      cp -v $src $out/newcob.val.tar.gz
+    '';
+  };
```

Comment:
Wait, isn't this just `fetchurl`?

Reply:
Ah, yes, that suffices! Thanks for the suggestions :heart: